docs/source/installation.rst

```diff
@@ -0,0 +1,90 @@
+.. _install:
+
+Install
+=======
+
+pygeoapi is easy to install on numerous environments.  Whether you are a user, administrator or developer, below
+are multiple approaches to getting pygeoapi up and running depending on your requirements.
+
+Requirements and dependencies
+-----------------------------
+
+pygeoapi runs on Python 3.
```

Comment:
pygeoapi won't build on python < 3.11 . We should indicate this on these instructions or else `python3 setup.py install` will fail.